public/app/plugins/datasource/postgres/partials/config.html

```diff
@@ -25,16 +25,17 @@ <h3 class="page-heading">PostgreSQL Connection</h3>
         on-reset="ctrl.onPasswordReset"
         on-change="ctrl.onPasswordChange"
         inputWidth="9"
+        aria-label="'Password'"
```

Comment:
I wonder if we need double quotes there?
```suggestion
        aria-label="Password"
```